### DISCUSSION TITLE
[Unstable] Mudraptor_unarmed instead of Mudraptor_unarmORed

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14655] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some events use the wrong enemy identifier

![image](https://github.com/user-attachments/assets/fb55fc0e-a0d8-41ce-8b06-4be3d3787cac)

![image](https://github.com/user-attachments/assets/34a6c056-380a-449d-8bd9-de9593169edc)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
The mudraptor population has been disarmed.

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in the next unstable build.
